code/modules/unit_tests/_map_correctness.dm

```diff
@@ -217,4 +218,15 @@ proc/check_turf_underlays()
 	if(log_msg)
 		CRASH(log_msg)
 
+proc/check_stacked_tables()
+	var/log_msg
+	for_by_tcl(table, /obj/table)
+		var/turf/T = table.loc
+		for (var/obj/table/other in T)
+			if (table != other && table.pixel_x == other.pixel_x && table.pixel_y == other.pixel_y) // pixel shifts okay
```

Comment:
Do I dare ask where there are pixel shifted tables stacked on one tile?

Reply:
`assets/maps/random_rooms/5x3/shadyoffice.dmm` not sure how to make this look similar without making it feel bad to run through. 

this still targets mistakes in map editing that are impossible to catch by eye/mapdiff; pixel shifts with different values should still be visible in the map editor and shouldn't look too odd in game. Granted, that's the only place that it's popping up for now.

Reply:
that's heinous but I won't make you fix it :P